client/my-sites/plans-v2/utils.ts

```diff
@@ -217,13 +231,92 @@ export function itemToSelectorProduct(
 			description: item.getDescription(),
 			monthlyProductSlug,
 			term: item.term === TERM_BIENNIALLY ? TERM_ANNUALLY : item.term,
-			features: { items: [] },
+			features: {
+				items: buildCardFeaturesFromItem( item ),
+				more: {
+					url: PLAN_COMPARISON_PAGE,
+					label: translate( 'See all features' ),
+				},
+			},
 			legacy: ! isResetPlan,
 		};
 	}
 	return null;
 }
 
+/**
+ * Builds a feature object of a product card, from a feature key.
+ *
+ * @param {JetpackPlanCardFeature} featureKey Key of the feature
+ * @returns {SelectorProductFeaturesItem} Feature item
+ */
+export function buildCardFeatureItemFromFeatureKey(
+	featureKey: JetpackPlanCardFeature
+): SelectorProductFeaturesItem | undefined {
+	let feature;
+	let subFeaturesKeys;
+
+	if ( isArray( featureKey ) ) {
+		const [ key, subKeys ] = featureKey;
```

Comment:
Was this meant to be something like

```js
const [ key, ...subKeys ] = featureKey;
```

The reason I ask is that below we assign `subFeaturesKeys = subKeys` and then call `subFeaturesKeys.map` but `String.prototype.map` is not a function, so this will always throw an error if that flow is followed.

Reply:
Most of the code in this utility is in fact deprecated. I did some clean up in #59577. Thanks for bringing this up!